### PR TITLE
Lowers portable pump max pressure to 1013.25

### PR DESCRIPTION
**Pull request by Shadowlight213**
Requested by @lzimann 
I think this is supposed to make bomb making harder or something, I don't remember exactly

:cl:
tweak: Portable pump max pressure has been lowered.
/:cl:

### CODE REVIEW

**Comment by duncathan** [comment]
why was it set to 30 atmospheres in the first place? nobody knows

**Comment by octareenroon91** [comment]
30 atmospheres happens to be a tank's safety limit.

There's a fixed supply of air pumps, so it's OK if they're a bit strong.

The expanded range probably has to do with the intended function of repressurizing rooms.

**Comment by Tokiko1** [comment]
Are singletanks common enough to warrant a nerf like this?

**Comment by Grazyn** [comment]
This kills 22/44/88 bombs. Giving those to miners was one of the few things I enjoyed about the game, expecially because I figured the procedure through trial and error.

**Comment by Grazyn** [comment]
Just tested it, this doesn't affect maxcaps, only makes it harder (or downright impossible) to make nukes for lavaland. I don't think it happened so often but at least now lavaland roles won't have to fear a shockwave hitting their ruin from miles away.

**Comment by CPTANT** [comment]
This also fucks over overfilling my jetpacks and supercharging the collectors.

**Comment by lzimann** [comment]
It is still possible to have huge bombs without this, also possible to have over filled jet packs.

**Comment by octareenroon91** [comment]
>It is still possible to have huge bombs without this, also possible to have over filled jet packs.

10x less convenient though, no?

**Comment by lzimann** [comment]
Yes, that's the whole point of the change. It's stupid having a 1 click 4 times the cap bomb.

**Comment by CPTANT** [comment]
It does extremely little to stop maxcap bombs, since just filling a regular tank with 10000 degree plasma and another with room temperature oxygen is already enough.....

Also I dont see how you can practically fill your jetpack after this.

**Comment by lzimann** [comment]
By using atmos.

**Comment by CPTANT** [comment]
O'rly. 

Seeing as the only interaction that jetpacks/plasma tanks/oxygen tanks have with any other system is through either a cannister airpump or scrubber the options you have are severly limited.

The only thing you can do is burn a mix while it is already inside the tank, but I wouldn't exactly call that practical cause above 3000 the tank starts to spill out burning plasma.

**Comment by lzimann** [comment]
Got this without using air pumps.

**Comment by lzimann** [comment]
Also:
```
ADMIN LOG: Bomb valve opened in  Atmospherics(JMP) with no device attacher: Unknown  Last touched by: Lzimann/(Monica Randolph)(?) (FLW)
ADMIN LOG: Explosion with size (16, 33, 66, 0) in area: Atmospherics (128,97,2) (JMP)
```
All without the use of 3k pressure air pumps.

**Comment by CPTANT** [comment]
If you can so easily get tanks to 3k pressure what is the point of this change to begin with?

**Comment by Grazyn** [comment]
Lzimann was upset that plebs were making fuckyuge bombs, bomb making is an elusive art that should remain in the hands of elder atmos masters.

But seriously this is probably for the best, maxcap making isn't touched, and all those new ruins like marine and dorfs won't get wrecked in the first 10 minutes by a random nuke.

**Comment by CPTANT** [comment]
I dislike it because it removes a lot of functionality from the airpump.

**Comment by lzimann** [comment]
It is not removing any functionality, only limiting them.

**Comment by CPTANT** [comment]
limiting to the point where the functionality isn't better than a cannister......

**Comment by lzimann** [comment]
A canister doesn't let you quickly remove all the gas of a tank like pumps do(any gas and not just harmful like scrubbers)

**Comment by Grazyn** [comment]
Also canister doesn't let you fill a tank to 1013 if it has been supercooled to the point of having really low pressure. Pump will still be useful for bomb making.

**Comment by CPTANT** [comment]
still removing functionality while doing nothing to stop people from getting maxcaps.

**Comment by duncathan** [comment]
this never should have been 30 atmospheres in the first place. 10 atmospheres should be the soft limit on portable atmos stuff like it is for canisters. jetpacks should be filled with supercooled gas anyway to maximize moles

**Comment by Lumipharon** [comment]
TBH the only actual issue here other than 'the plebs can do the elder tier shit, reeee', is easily nuking lavaland. 
If being able to nuke lavaland is an issue then just like, change that?

**Comment by Grazyn** [comment]
>If being able to nuke lavaland is an issue then just like, change that?
If you lower the maxcap on lavaland you won't be able to show off your throbbing bomb-making e-peen.

>this never should have been 30 atmospheres in the first place. 
Tanks can hold a maximum of 30-something atmospheres before leaking, the pump just allowed you to fill a tank to the max.

**Comment by CPTANT** [comment]
Merging your own Pr's isn't allowed man.

**Comment by Shadowlight213** [comment]
It's the loophole. He didn't make the pr

**Comment by jughu** [comment]
>2 upvoted
>12 downvoted

Merged anyways

**Comment by CPTANT** [comment]
@jughu 

what do you expect when the person that requests the change also merges it.

**Comment by Hanador** [comment]
I'm not a coder merely a humble bomb maker/atmosian. I can understand the proposed change but I disagree with it.
Air pumps had two real roles and this fucks over both of them.
1. Atmos' Refilling/draining a room with air.
2. Filling tanks (oxygen and plasma from the dispenser) to the maximum allowed kPa.
This significantly weakens/removes the point of both uses for pumps and doesn't add anything constructive or fun to the game. While you can use air scrubbers or vents most of the time, if the power is out you are forced to use pumps. As for filling the tanks up this is a huge nerf to lots of creative traitoring (power-fisting and that other secret thing) and large bomb making. As it stands now even if the change is implemented it does nothing to stop large bomb making as the people who bothered to learn how to make large explosions can do the exact same thing but with one additional step (starting a fire in the tank to increase the pressure). If you are so worried about lavaland being nuked then put into place the same bombcap that exists on station or make the ruins bombproof or something.


TL;DR this is kinda stupid for utility purposes (engine, atmos, bombs)
 
 Also Lzi I fucked up that one day the third screenshot is the one that would have gone off if I had not flubbed the mix (I mistook the third mix for the one I used in screenshot 2)
screenshot 1
screenshot 2
screenshot 3
